### PR TITLE
Fuzz the Table from JS

### PR DESCRIPTION
**Pull request by kripken**
Continues the work from #7027 which added throwing from JS, this adds
table get/set operations from JS, to further increase our coverage of
Wasm/JS interactions (the table can be used from both sides).

### CODE REVIEW

File: src/tools/fuzzing/fuzzing.cpp [hunk 1]
```diff
@@ -609,6 +612,49 @@ void TranslateToFuzzReader::addImportThrowingSupport() {
...
+  // multiple ones TODO
+  if (!funcrefTableName) {
+    return;
+  }
+
+  // If a "table" export already exists, skip fuzzing these imports, as the
+  // current export may not contain a valid table for it.
+  if (wasm.getExportOrNull("table")) {
+    return;
+  }
```
Comment (aheejin) [inline src/tools/fuzzing/fuzzing.cpp:628]: In which case can this happen?

File: scripts/fuzz_shell.js [hunk 2]
```diff
@@ -152,7 +152,15 @@ var imports = {
     // Throw an exception from JS.
     'throw': () => {
       throw 'some JS error';
-    }
+    },
+
+    // Table operations.
+    'table-get': (index) => {
+      return exports.table.get(index >>> 0);
```
Comment (aheejin) [inline scripts/fuzz_shell.js:159]: What's the reason for shifting the index by 0? (I googled it and it says it makes it a number, but I don't understand why it is necessary...)

**Review by aheejin — APPROVED** [review]

File: src/tools/fuzzing/fuzzing.cpp (hunk 1 above)
Comment (kripken) [reply src/tools/fuzzing/fuzzing.cpp:628]: The fuzzer can start with inputs from the test suite and elsewhere, and initial content like that can contain a "table" export. Basically, the fuzzer needs to be able to start from valid wasm file, so we have to handle corner cases like that.

File: scripts/fuzz_shell.js (hunk 2 above)
Comment (kripken) [reply scripts/fuzz_shell.js:159]: the `>>>` operator is an unsigned operation, so this forces it to be a 32-bit unsigned number. That is the same way that the wasm `table.get` operation treats the input, so this is done to match that.

**Review by aheejin — APPROVED** [review]
